apps/src/codebridge/types.ts

```diff
@@ -57,7 +57,7 @@ export type ConfigType = {
 export type ProjectType = MultiFileSource;
 export type SourceType = ProjectSources;
 
-export type SetProjectFunction = (project: ProjectType) => void;
+export type SetProjectFunction = (project: SourceType) => void;
```

Comment:
Now that `labConfig` is being saved as part of the project, I find these two type names to be a little confusing, almost flip-flopped.
I would assume `SourceType` is a `MultiFileSource`, and `ProjectType` to be `ProjectSources`.